app/build.gradle

```diff
@@ -27,8 +27,8 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile project(':lock-facebook')
-    compile 'com.auth0.android:lock:2.4.0'
+    compile 'com.auth0.android:lock:2.5.0'
     compile 'com.github.bumptech.glide:glide:3.7.0'
-    compile 'com.android.support:support-v4:25.1.0'
-    compile 'com.android.support:appcompat-v7:25.1.0'
+    compile 'com.android.support:support-v4:25.0.0'
```

Comment:
AppCompat versions must be the same for all dependencies to avoid conflicts. In this case, FacebookSDK sets the `25.0.0`. See http://stackoverflow.com/questions/42374151/all-com-android-support-libraries-must-use-the-exact-same-version-specification/42589182